lib/validator.js

```diff
@@ -338,7 +338,6 @@ Specberus.prototype.getFeedbackDueDate = function() {
         if (candidates !== null) {
             for (var i = 0; i < candidates.length; i++) {
             var d = this.stringToDate(candidates[i]);
-            dates.list.push(d);
```

Comment:
dates.list is still being used.

https://github.com/w3c/specberus/blob/b604251e2eb2303227e30c1dce282da2aa410643/lib/rules/sotd/cr-end.js#L28-L30